File: Day08/Easy/Solution.py
class Solution:
    def reverseArray(self, arr):
        i, j = 0, len(arr)-1
        while i < j:
            arr[i], arr[j] = arr[j], arr[i]
            i += 1
            j -= 1

if __name__ == "__main__":
    sol = Solution()
    arr = [1, 2, 3, 4, 5]
    sol.reverseArray(arr)
    print("Reversed array:", arr)

File: Day08/Easy/Solution.cpp
#include <bits/stdc++.h>
using namespace std;

class Solution {
public:
    void reverseArray(vector<int> &arr) {
        int n = arr.size();
        int i = 0, j = n-1;
        while(i < j){
            swap(arr[i], arr[j]);
            i++;
            j--;
        }
    }
};

int main() {
    Solution sol;
    vector<int> arr = {1, 2, 3, 4, 5};
    sol.reverseArray(arr);

    cout << "Reversed array: ";
    for(int x : arr) cout << x << " ";
    cout << endl;
    return 0;
}
